cmd/kops/export_kubecfg.go

```diff
@@ -71,6 +73,8 @@ func NewCmdExportKubecfg(f *util.Factory, out io.Writer) *cobra.Command {
 
 	cmd.Flags().StringVar(&options.KubeConfigPath, "kubeconfig", options.KubeConfigPath, "The location of the kubeconfig file to create.")
 	cmd.Flags().BoolVar(&options.all, "all", options.all, "export all clusters from the kops state store")
+	cmd.Flags().BoolVar(&options.admin, "admin", options.admin, "also export the admin user")
```

Comment:
```suggestion
	cmd.Flags().BoolVar(&options.admin, "admin", options.admin, "export the admin user")
```

```suggestion
	cmd.Flags().BoolVar(&options.admin, "admin", options.admin, "export the admin user and use in the cluster context")
```

maybe one of these? I'll keep thinking of how this could be worded...

Reply:
"include the admin user and credentials"